### PR TITLE
Polls 05 30

### PR DESCRIPTION
**Pull request by Hipogeo**
@TheExistence

 There you go, Joa! :)

### CODE REVIEW

File: governance/polls/Offboarding Parameters for UNI-A - May 30, 2022.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+The community can vote in this poll to express support or opposition to the following changes: 
+* Implementing the following risk parameters for offboarding UNI-A:
+
+**Proposed Offboarding Parameters**
+
+* Enable liquidations for UNI-A
+* Set [Liquidation Penalty](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%
+* Set [Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam) to 0
+* Set [tip](https://manual.makerdao.com/parameter-index/collateral-auction/param-flat-kick-incentive) to 0
+* Set duration of [dss-lerp](https://manual.makerdao.com/module-index/module-lerp) for liquidation ratio to 30 days
```
Comment (TheExistence) [inline governance/polls/Offboarding Parameters for UNI-A - May 30, 2022.md:36]: Please change (dss-lerp) for Linear Interpolation Module (dss-lerp)

File: governance/polls/Offboarding Parameters for UNIV2DAIETH-A - May 30, 2022.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+The community can vote in this poll to express support or opposition to the following changes: 
+* Implementing the following risk parameters for offboarding UNIV2DAIETH-A:
+
+**Proposed Offboarding Parameters**
+
+* Enable liquidations for UNIV2DAIETH-A
+* Set [Liquidation Penalty](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%
+* Set [Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam) to 0
+* Set [tip](https://manual.makerdao.com/parameter-index/collateral-auction/param-flat-kick-incentive) to 0
+* Set duration of [dss-lerp](https://manual.makerdao.com/module-index/module-lerp) for liquidation ratio to 30 days
```
Comment (TheExistence) [inline governance/polls/Offboarding Parameters for UNIV2DAIETH-A - May 30, 2022.md:36]: Please change (dss-lerp) for Linear Interpolation Module (dss-lerp)

File: governance/polls/Offboarding Parameters for UNIV2UNIETH-A - May 30, 2022.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+The community can vote in this poll to express support or opposition to the following changes: 
+* Implementing the following risk parameters for offboarding UNIV2UNIETH-A:
+
+**Proposed Offboarding Parameters**
+
+* Enable liquidations for UNIV2UNIETH-A
+* Set [Liquidation Penalty](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%
+* Set [Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam) to 0
+* Set [tip](https://manual.makerdao.com/parameter-index/collateral-auction/param-flat-kick-incentive) to 0
+* Set duration of [dss-lerp](https://manual.makerdao.com/module-index/module-lerp) for liquidation ratio to 30 days
```
Comment (TheExistence) [inline governance/polls/Offboarding Parameters for UNIV2UNIETH-A - May 30, 2022.md:36]: Please change (dss-lerp) for Linear Interpolation Module (dss-lerp)

File: governance/polls/Offboarding Parameters for UNIV2WBTCDAI-A - May 30, 2022.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+The community can vote in this poll to express support or opposition to the following changes: 
+* Implementing the following risk parameters for offboarding UNIV2WBTCDAI-A:
+
+**Proposed Offboarding Parameters**
+
+* Enable liquidations for UNIV2WBTCDAI-A
+* Set [Liquidation Penalty](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%
+* Set [Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam) to 0
+* Set [tip](https://manual.makerdao.com/parameter-index/collateral-auction/param-flat-kick-incentive) to 0
+* Set duration of [dss-lerp](https://manual.makerdao.com/module-index/module-lerp) for liquidation ratio to 30 days
```
Comment (TheExistence) [inline governance/polls/Offboarding Parameters for UNIV2WBTCDAI-A - May 30, 2022.md:36]: Please change (dss-lerp) for Linear Interpolation Module (dss-lerp)

File: governance/polls/Offboarding Parameters for UNIV2WBTCETH-A - May 30, 2022.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+The community can vote in this poll to express support or opposition to the following changes: 
+* Implementing the following risk parameters for offboarding UNIV2WBTCETH-A:
+
+**Proposed Offboarding Parameters**
+
+* Enable liquidations for UNIV2WBTCETH-A
+* Set [Liquidation Penalty](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%
+* Set [Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam) to 0
+* Set [tip](https://manual.makerdao.com/parameter-index/collateral-auction/param-flat-kick-incentive) to 0
+* Set duration of [dss-lerp](https://manual.makerdao.com/module-index/module-lerp) for liquidation ratio to 30 days
```
Comment (TheExistence) [inline governance/polls/Offboarding Parameters for UNIV2WBTCETH-A - May 30, 2022.md:36]: Please change (dss-lerp) for Linear Interpolation Module (dss-lerp)

File: governance/polls/Offboarding Parameters for UNI-A - May 30, 2022.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
+
+---
```
[the remark this answers is not in the document]
Comment (Hipogeo) [reply governance/polls/Offboarding Parameters for UNI-A - May 30, 2022.md:2]: Fixed!

File: governance/polls/Offboarding Parameters for UNI-A - May 30, 2022.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+title: Offboarding Parameters for UNI-A - May 30, 2022
+summary: Signal your support or opposition to the following liquidation parameters, designed to offboard UNI
+discussion_link: https://forum.makerdao.com/t/signal-request-offboard-uni-univ2daieth-univ2wbtceth-univ2unieth-and-univ2wbtcdai/15160
+vote_type: Plurality Voting
+categories:
```
[the remark this answers is not in the document]
Comment (Hipogeo) [reply governance/polls/Offboarding Parameters for UNI-A - May 30, 2022.md:7]: Fixed!

File: governance/polls/Offboarding Parameters for UNI-A - May 30, 2022.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes: 
+* Implementing the following risk parameters for offboarding UNI-A:
```
[the remark this answers is not in the document]
Comment (Hipogeo) [reply governance/polls/Offboarding Parameters for UNI-A - May 30, 2022.md:28]: Fixed!

File: governance/polls/Offboarding Parameters for UNI-A - May 30, 2022.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+**Proposed Offboarding Parameters**
+
+* Enable liquidations for UNI-A
+* Set [Liquidation Penalty](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%
```
[the remark this answers is not in the document]
Comment (Hipogeo) [reply governance/polls/Offboarding Parameters for UNI-A - May 30, 2022.md:33]: Fixed!

File: governance/polls/Offboarding Parameters for UNIV2DAIETH-A - May 30, 2022.md [hunk 10]
```diff
@@ -0,0 +1,59 @@
+
+---
```
[the remark this answers is not in the document]
Comment (Hipogeo) [reply governance/polls/Offboarding Parameters for UNIV2DAIETH-A - May 30, 2022.md:1]: Fixed!

File: governance/polls/Offboarding Parameters for UNI-A - May 30, 2022.md (hunk 1 above)
Comment (Hipogeo) [reply governance/polls/Offboarding Parameters for UNI-A - May 30, 2022.md]: Fixed on all!

File: governance/polls/Offboarding Parameters for UNIV2DAIETH-A - May 30, 2022.md (hunk 2 above)
Comment (Hipogeo) [reply governance/polls/Offboarding Parameters for UNIV2DAIETH-A - May 30, 2022.md]: Fixed on all!

File: governance/polls/Offboarding Parameters for UNIV2UNIETH-A - May 30, 2022.md (hunk 3 above)
Comment (Hipogeo) [reply governance/polls/Offboarding Parameters for UNIV2UNIETH-A - May 30, 2022.md]: Fixed on all!

File: governance/polls/Offboarding Parameters for UNIV2WBTCDAI-A - May 30, 2022.md (hunk 4 above)
Comment (Hipogeo) [reply governance/polls/Offboarding Parameters for UNIV2WBTCDAI-A - May 30, 2022.md]: Fixed on all!

File: governance/polls/Offboarding Parameters for UNIV2WBTCETH-A - May 30, 2022.md (hunk 5 above)
Comment (Hipogeo) [reply governance/polls/Offboarding Parameters for UNIV2WBTCETH-A - May 30, 2022.md]: Fixed on all!

**Review by TheExistence — APPROVED** [review]
LGTM @prose11 Thanks @Hipogeo :)

**Pull request merged**